resource-managers/kubernetes/integration-tests/src/test/scala/org/apache/spark/deploy/kubernetes/integrationtest/KubernetesSuite.scala

```diff
@@ -226,6 +226,26 @@ private[spark] class KubernetesSuite extends SparkFunSuite with BeforeAndAfter {
         Seq.empty[String])
   }
 
+  test("Setting JVM options on the driver and executors with spaces.") {
```

Comment:
I'm ok with keeping the name as-is for now though given that the next PR is imminent.